.travis.yml

```diff
@@ -16,10 +17,16 @@ before_install:
       cd ..;
     fi
 install:
+  - npm install -g elm@0.19.0-bugfix2 elm-test@0.19.0-beta8 elm-format
```

Comment:
The key is here: for some reason right now when installing `elm-test` it installs the 0.18 version, when and the version `0.19.0-beta9` is broken: https://github.com/rtfeldman/node-test-runner/issues/288#issuecomment-423619025